sound/soc/sof/intel/hda.c

```diff
@@ -297,6 +307,268 @@ static const struct sof_intel_dsp_desc *get_chip_info(int pci_id)
 	return NULL;
 }
 
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
You probably want to rename this Kconfig to CONFIG_SND_SOC_SOC_HDA_LINK ?
HDA can mean controller, bus or codec. Here you handle the two latter topics.

Reply:
good suggestion, will change it.